dnf5/commands/clean/clean.cpp

```diff
@@ -116,8 +116,9 @@ void CleanCommand::run() {
     auto & ctx = get_context();
     fs::path cachedir{ctx.base.get_config().cachedir().get_value()};
 
+    std::error_code ec;
     libdnf::repo::RepoCache::RemoveStatistics statistics{};
-    for (const auto & dir_entry : std::filesystem::directory_iterator(cachedir)) {
+    for (const auto & dir_entry : std::filesystem::directory_iterator(cachedir, ec)) {
```

Comment:
Is it intended, that issues are silent?

Reply:
In most cases yes. But thinking about this particular case - we can print a warning to std::err.